sionna/mimo/equalization.py

```diff
@@ -139,6 +140,9 @@ def lmmse_equalizer(y, h, s, whiten_interference=True):
     # Compute residual error variance
     one = tf.cast(1, dtype=d.dtype)
     no_eff = tf.math.real(one/d - one)
+    # Deal with negative or zero values due to numerical error.
+    eps = np.finfo(d.dtype.as_numpy_dtype).eps
```

Comment:
It is not exactly clear to me why this is needed here. The calculation of `no_eff` does not look unstable.
I see that `no_eff` might become 0, however, this should be captured by the updated demapper, right?

I suggest to remove the updates on `equalizer.py`.

Reply:
> I see that no_eff might become 0, however, this should be captured by the updated demapper, right?

I also thought that, but IIRC I got negative values here with the PUSCH decoder. I will double check and see if I can find an easy way to reproduce this.

Reply:
I'll leave this out of this PR and will open another one when I get to reproduce this.